tools/serverless-plugin/src/generators/service/general-files/serverless.yml.template

```diff
@@ -25,6 +26,9 @@ provider:
   stage: ${opt:stage, 'dev'}
   region: ${opt:region, 'ap-southeast-2'}
   memorySize: 192 #mb (default for all)
+  tracing:
+    apiGateway: true
+    lambda: true
```

Comment:
Doesn't this require the tracing plugin to work?

Reply:
@ryanrixxh This enables X-ray tracing in the infrastructure layer. For X-ray to work, we need both this configuration & capturing the HTTP requests (in our code).